webcompat/dashboard.py

```diff
@@ -42,17 +42,19 @@ def filter_needstriage(milestone_list):
                                     for issue in needstriage_list
                                     if issue['older'] is True])
     dashboard_stats['needsinfo'] = len([issue['needsinfo']
-                                       for issue in needstriage_list
-                                       if issue['needsinfo'] is True])
+                                        for issue in needstriage_list
+                                        if issue['needsinfo'] is True])
     return needstriage_list, dashboard_stats
 
 
 def has_needsinfo(labels):
     """Assess if the issue has a needsinfo label."""
-    needsinfo = False
-    if 'status-needsinfo' in labels:
-        needsinfo = True
-    return needsinfo
+    print(labels)
+    needsinfo = (label for label in labels if label.startswith(
+        'status-needsinfo'))
```

Comment:
nit: not very important, but this is probably more readable.

```python
needsinfo = (label for label in labels 
             if label.startswith('status-needsinfo'))
```

Reply:
I agree